Dockerfile.antidote

```diff
@@ -0,0 +1,113 @@
+ARG ELIXIR_VERSION=1.13.4
+ARG OTP_VERSION=24.3
+ARG DEBIAN_VERSION=bullseye-20210902-slim
+
+ARG BUILDER_IMAGE="hexpm/elixir:${ELIXIR_VERSION}-erlang-${OTP_VERSION}-debian-${DEBIAN_VERSION}"
+ARG RUNNER_IMAGE="debian:${DEBIAN_VERSION}"
+
+FROM ${BUILDER_IMAGE} AS builder
+
+LABEL maintainer="dev@vaxine.io"
+
+RUN apt-get update -y && apt-get install -y build-essential git curl \
+    && apt-get clean && rm -f /var/lib/apt/lists/*_*
+
+RUN mix local.hex --force                                                                                                                                                
+RUN mix local.rebar --force  
+
+WORKDIR /app
+
+ENV CI=true
+
+COPY rebar.config /app/
+COPY rebar.lock /app/
+
+RUN /root/.mix/rebar3 compile
+
+COPY Makefile /app/
+COPY config/ /app/config/
+COPY apps/ /app/apps/
+
+RUN /root/.mix/rebar3 compile
+RUN /root/.mix/rebar3 release
+
+FROM ${RUNNER_IMAGE} AS runner_setup
+
+RUN apt-get update -y && apt-get install -y libstdc++6 openssl libncurses5 locales \
+  && apt-get clean && rm -f /var/lib/apt/lists/*_*
+
+# Set the locale
+RUN sed -i '/en_US.UTF-8/s/^# //g' /etc/locale.gen && locale-gen
+
+ENV LANG en_US.UTF-8
+ENV LANGUAGE en_US:en
+ENV LC_ALL en_US.UTF-8
+
+WORKDIR "/app"
+RUN chown nobody /app
+
+# set runner ENV
+ENV MIX_ENV="prod"
+
+
+FROM runner_setup AS runner
+
+## Antidote configuration via environment variables
+COPY --from=builder --chown=nobody:root /app/_build/default/rel/antidote ./
+
+### Data storage
+# where the logs and data are stored and exposed
+ENV ROOT_DIR_PREFIX="/antidote-data/" \
+    DATA_DIR_PREFIX="data/" \
+    LOGGER_DIR_PREFIX="log/" 
+RUN mkdir -p ${ROOT_DIR_PREFIX}
+RUN chown nobody ${ROOT_DIR_PREFIX}
+
+ENV \
+    ### Riak
+    HANDOFF_PORT="8099" \
+    RING_SIZE="16" \
+    ### Inter-DC communication
+    LOGREADER_PORT="8085" \
+    PBSUB_PORT="8086" \
+    ### Erlang
+    NODE_NAME="antidote@antidote" \
+    COOKIE="secret"\
+    ### Logger verbosity: debug, info, notice, warning, error, alert, critical
+    DEBUG_LOGGER_LEVEL="info"
+    
+
+### Public client
+ENV PB_PORT "8087"
+EXPOSE ${PB_PORT}
+
+## Antidote Features Configuration
+##   Adjusting features for a node restart can have unexpected consequences
+##   Decide on features before first starting and operating the AntidoteDB node
+##   see config/sys.config.src for documentation
+ENV ANTIDOTE_TXN_CERT="true" \
+    ANTIDOTE_TXN_PROT="clocksi" \
+    ANTIDOTE_RECOVER_FROM_LOG="true" \
+    ANTIDOTE_META_DATA_ON_START="true" \
+    ANTIDOTE_SYNC_LOG="false" \
```

Comment:
This should be `true`

Reply:
The settings are copied verbatim from https://github.com/AntidoteDB/docker-antidote/blob/master/latest/Dockerfile. This PR is meant to allow us to build locker Antidote code - which is a more common pattern - instead of relying on in-docker checkouts like the previous one.

I'll address updated ENV variables for production in actual production configs in k8s :)

Reply:
Got it :)